internal/protoschema/jsonschema/jsonschema.go

```diff
@@ -501,8 +501,11 @@ func generateIntValidation[T int32 | int64](
 			isOr = rules.GetLte() <= rules.GetGt()
 		}
 		if isOr {
-			orNumberSchema = map[string]any{"exclusiveMinimum": rules.GetGt()}
-		} else {
+			orNumberSchema = make(map[string]any)
+			if int64(rules.GetGt()) >= jsMinInt {
```

Comment:
Is the goal to not emit any min/max constraints outside `[jsMinInt,jsMaxInt]`? If so don't we need to ensure this value is `<= jsMaxInt` here too?

Reply:
Think of 'integer' (not 'number') as having an implicit range of [jsMinInt, jsMaxInt]. This makes not emitting limits outside this range a noop. If the number must be greater than or less than this limit, then no 'integer' is valid (only string integers). I'd rather not remove the limit in this case, but I added a TODO to not emit the entire number option.